posttest.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+require('./test/init.js');
+
+var async = require('async');
+var _ = require('lodash');
+
+var db = global.getDataSource();
+var schemaName = global.config.schema;
+var findTables = 'SELECT TABNAME FROM SYSCAT.TABLES WHERE TABSCHEMA = ' +
+  '\'' + schemaName + '\'' + ' and type=\'T\'' + ';';
+var findViews = 'SELECT TABNAME FROM SYSCAT.TABLES WHERE TABSCHEMA = ' +
+  '\'' + schemaName + '\'' + ' and type=\'V\'' + ';';
+
+if (!process.env.CI)
+  return console.log('Skipping post-test ...');
+console.log('> Cleaning up the database ...');
+
+// drop all the tables and views from the schema
+db.connector.execute(findViews, function(err, views) {
+  if (err) throw err;
+
+  async.each(views, function(view, callback) {
+    var viewName = view.TABNAME;
+    db.connector.execute('DROP VIEW "' + schemaName + '"."' +
+      viewName + '";', function(err, result) {
+      callback(err);
+    });
+  }, function(err) {
```

Comment:
I am not sure what you mean. This is the `posttest` file. It drops all the views, then drops the tables and then finally drops the schema.